account_promissory_note_bankia/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Impresión de pagaré Bankia A4",
+    "version": "13.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "Tecnativa, Odoo Community Association (OCA)",
+    "category": "Accounting",
+    "website": "https://github.com/OCA/l10n-spain",
+    "depends": [
+        "account_check_printing_report_base",
```

Comment:
`account_check_printing_report_base` + `account_payment_promissory_note` addons exist in https://github.com/OCA/account-payment and need to update `oca_dependencies.txt` to add **account-payment** to load addons correctly and probably Travis will solved